assets/js/base/components/quantity-selector/index.js

```diff
@@ -16,20 +17,16 @@ import './style.scss';
 const QuantitySelector = ( {
 	className,
 	quantity = 1,
+	minimum = 1,
+	maximum = undefined,
 	onChange = () => null,
 	itemName = '',
 	disabled,
 } ) => {
 	const classes = classNames( 'wc-block-quantity-selector', className );
-	const inputRef = useRef( null );
-	const [ currentValue, setCurrentValue ] = useState( quantity );
 
-	useEffect( () => {
-		if ( currentValue !== inputRef.current.value ) {
-			inputRef.current.value = currentValue;
-		}
-		onChange( currentValue );
-	}, [ currentValue, onChange ] );
+	const canDecrease = quantity > minimum;
+	const canIncrease = quantity < maximum || isUndefined( maximum );
```

Comment:
This should be the other way around:

```suggestion
	const canIncrease = typeof maximum === 'undefined' || quantity < maximum;
```

Reply:
I'm pretty sure these are correct, they work well when testing - am I missing something?

![increase-decrease](https://user-images.githubusercontent.com/4167300/75714866-563f6f80-5d31-11ea-87b7-462e171fac1f.gif)

(Note in this gif `Album` and `Hoodie with Logo` are limited to one per cart so both buttons are disabled, `canDecrease` == `canIncrease` == false. Middle product allows multiple per cart, and correctly limits > 1.)

Reply:
What I meant was the the undefined check should come before the comparison. It's not a breaking thing, just good habit to do least expensive to most expensive in expressions (granted this case is likely negligible , but still _habits_ ;) ) I see you switched the order anyways in your latest commit 👍